Album/ViewControllers/MapControllers/FriendMapViewController.m

```diff
@@ -51,46 +55,47 @@ - (void)loadView {
     // Initialize data structures to cache retrieved data
     self.placeToPins = [[NSMutableDictionary alloc] init];
     self.pinImages = [[NSMutableDictionary alloc] init];
+} /* loadView */
 
-}
-- (void) loadMarkers {
+- (void)loadMarkers {
     // Place markers on initial map view
-    int i=0;
-    while(i<self.markerArr.count) {
-        Pin *pin=self.markerArr[i];
+    int i = 0;
+    while (i < self.markerArr.count) {
+        Pin *pin = self.markerArr[i];
         GMSMarker *marker = [[GMSMarker alloc] init];
         marker.position = CLLocationCoordinate2DMake(pin.latitude, pin.longitude);
         marker.title = pin.placeName;
+        marker.icon = [GMSMarker markerImageWithColor:[self.colorHelper colorFromHexString:self.user[@"colorHexString"]]];
         marker.snippet = pin.placeID;
         marker.map = self.mapView;
         i++;
     }
 }
 
-- (void) fetchMarkers {
+- (void)fetchMarkers {
     // Query to find markers that belong to specific user
     PFQuery *query = [PFQuery queryWithClassName:classNamePin];
     [query whereKey:@"author" equalTo:self.user];
-    [query includeKey: @"objectId"];
+    [query includeKey:@"objectId"];
     [query findObjectsInBackgroundWithBlock:^(NSArray *pins, NSError *error) {
-             if (pins != nil) {
-             // Store the posts, update count
-             NSLog(@"Successfully fetched markers!");
-             self.markerArr = (NSMutableArray *)pins;
-             [self loadMarkers];
-         } else {
-             NSLog(@"%@", error.localizedDescription);
-         }
-     }];
+               if (pins != nil) {
+               // Store the posts, update count
+               NSLog(@"Successfully fetched markers!");
+               self.markerArr = (NSMutableArray *)pins;
+               [self loadMarkers];
+               } else {
+               NSLog(@"%@", error.localizedDescription);
```

Comment:
other than logging an error, consider letting the user know that an error happened

Reply:
Added alert!